iree/tools/BUILD

```diff
@@ -300,6 +300,7 @@ cc_binary(
     tags = ["hostonly"],
     deps = [
         "//build_tools:default_linkopts",
+        "//build_tools:threads",
```

Comment:
We _could_ add a TODO here to remove this and unify with CMake once TF uses the same BUILD file for LLVM as we do.
Targets here in `tools/` tend to grow all sorts of deps though, so we can just look at that as part of general cleanup once in a while.